### ISSUE TITLE
My Deeplab model trained on my own dataset behaves unexpecetedly

### ISSUE THREAD

**Issue by vipghn4** [tensorflow/models#6887]
I've trained DeepLab model on my own dataset for 170,000 iterations using the command

```
!python deeplab/train.py \
    --logtostderr \
    --training_number_of_steps=170000 \
    --last_layers_contain_logits_only=False \
    --initialize_last_layer=True \
    --base_learning_rate=1e-3 \
    --slow_start_step=1000 \
    --slow_start_learning_rate=3e-4\
    --train_split="train" \
    --model_variant="mobilenet_v2" \
    --output_stride=16 \
    --train_crop_size="613, 613" \
    --train_batch_size=8 \
    --fine_tune_batch_norm=true \
    --dataset="clothes" \
    --tf_initial_checkpoint="./models/research/deeplab/exp/clothes_train/train/model.ckpt" \
    --train_logdir="./models/research/deeplab/exp/clothes_train/train" \
    --dataset_dir="./tfrecord"
```

When I fine tune my model with the command

```
!python deeplab/train.py \
    --logtostderr \
    --training_number_of_steps=3000 \
    --train_split="train" \
    --model_variant="mobilenet_v2" \
    --output_stride=16 \
    --train_crop_size="613,613" \
    --train_batch_size=8 \
    --base_learning_rate=3e-7 \
    --dataset="clothes" \
    --initialize_last_layer \
    --tf_initial_checkpoint="./models/research/deeplab/exp/clothes_train/train/model.ckpt-170000" \
    --train_logdir="./models/research/deeplab/exp/clothes_train/train" \
    --dataset_dir="./tfrecord"
```

The loss appears as if my model hasn't been trained at all.

```
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Saving checkpoints for 0 into /content/drive/My Drive/Projects/ClothesSegmentation/models/research/deeplab/exp/clothes_train/quantized_train/model.ckpt.
Total loss is :[3.09079671]
```

I've tried to freeze my old model (i.e. `model.ckpt-170000`) and it performs well in `deeplab_demo.ipynb`. I also modified the `tf_initial_checkpoint` in my fine-tuning command from `./models/research/deeplab/exp/clothes_train/train/model.ckpt-170000` to `./models/research/deeplab/exp/clothes_train/train/model.ckpt` and the fine-tuning process actually works well (i.e. it takes my trained model and continues training). I also checked the output given by `model.ckpt-170000` by converting the model into a TFLite model and plot the output segmentation but the output are all zeros (i.e. black image).

Can anyone explain it for me ?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
